websites/serializers.py

```diff
@@ -263,6 +257,10 @@ def get_draft_url(self, instance):
         """Get the draft url for the site"""
         return instance.get_full_url(version=VERSION_DRAFT)
 
+    def get_has_site_metadata(self, instance):
+        site_metadata = instance.websitecontent_set.filter(type="sitemetadata")
+        return True if (site_metadata and site_metadata.metadata) else False
```

Comment:
Your code is correct. In my opinion, we can make it easier to read too.

```suggestion
        return bool(site_metadata and site_metadata.metadata)
```